core/grants.go

```diff
@@ -90,6 +97,7 @@ func (a *API) createGrant(ctx context.Context, x apiGrant) (*apiGrant, error) {
 		GuardData: data,
 		Policy:    g.Policy,
 		CreatedAt: g.CreatedAt,
+		Protected: g.Protected,
```

Comment:
should we include this when creating a grant? it will (probably) always be false.